app/controllers/report_controller/schedules.rb

```diff
@@ -109,7 +109,7 @@ def miq_report_schedule_run_now
             else
               _("The selected Schedules have been queued to run")
             end
-      add_flash(msg, :info, true)
+      add_flash(msg, :success, true)
```

Comment:
Is this really a part of the PR's context? Why are you changing this?

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/5923#discussion_r310997568

Reply:
Good enough